### ISSUE TITLE
No translator selection bar in parallel translator mode 

### ISSUE THREAD

**Issue by czarnyckm** [you-apps/TranslateYou#454]
### Steps to reproduce

1. Enable parallel translation
2. Enable all translators provider for parallel translate
3. On main screen the bottom translate provider selection bar is not shown

### Expected behavior

The translate provider selection bar should be shown on the main screen like in the version 9.0

### Actual behavior

In parallel translate mode no longer the provider selection bar is shown

### Translate You version

10.0

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Duplicate of https://github.com/you-apps/TranslateYou/issues/448, will be fixed with the next release
